rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2021/01/09"
+maturity = "development"
+updated_date = "2021/01/09"
+
+[rule]
+author = ["Skoetting"]
+description = "User Added to Privileged Group in Active Directory"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
```

We tend to extend endpoint events lookback a bit further back to account for a larger buffer window for sending events

```suggestion
from = "now-9m"
index = ["winlogbeat-*", "logs-endpoint.events.*"]
```